src/status_im2/contexts/activity_center/notification/admin/view.cljs

```diff
@@ -0,0 +1,55 @@
+(ns status-im2.contexts.activity-center.notification.admin.view
+  (:require [i18n.i18n :as i18n]
+            [quo2.core :as quo]
+            [quo2.foundations.colors :as colors]
+            [status-im.constants :as constants]
+            [status-im2.contexts.activity-center.notification.common.style :as style]
+            [status-im2.contexts.activity-center.notification.common.view :as common]
+            [utils.datetime :as datetime]
+            [utils.re-frame :as rf]))
+
+(defn view
+  [{:keys [author communityId id membershipStatus read timestamp]}]
```

Comment:
We could update the function `<-rpc` to also rename the keys `communityId` and `membershipStatus`, which are using camel case.

Please see https://github.com/status-im/status-mobile/blob/develop/src/status_im/data_store/activities.cljs#L36

Reply:
Updated 👍